### PR TITLE
Hotfix/middle table multitenancy

### PR DESCRIPTION
**Pull request by basil7vals**

### CODE REVIEW

File: app/models/issue.rb [hunk 1]
```diff
@@ -77,4 +75,8 @@ def send_email
       IssueMailer.delay.notify(email, id, company_id)
     end
   end
+
+  def issue_watchers_count
+    issue_watcher.count
```
Comment (abubakarazeem) [inline app/models/issue.rb]: remove this

File: app/views/issues/show.html.erb [hunk 2]
```diff
@@ -20,7 +20,7 @@
   </div>
 
   <% if can?(:destroy, @issue) %>
-    <%= link_to "Delete", project_issue_path(@issue.project_id, @issue.id), method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-secondary float-right mr-2' %>
+    <%= link_to "Delete", project_issue_path(@issue.project_id, @issue.id), method: :delete, data: { confirm: issue_delete_data_confirm(@issue.issue_watchers.count) }, class: 'btn btn-secondary float-right mr-2', id: 'issue_delete_button' %>
```
Comment (abubakarazeem) [inline app/views/issues/show.html.erb]: count query should be in `issue_delete_data_confirm` helper method

**Comment by abubakarazeem** [comment]
review: cycle 1

File: app/models/issue_watcher.rb [hunk 3]
```diff
@@ -1,7 +1,6 @@
 class IssueWatcher < ActiveRecord::Base
-  not_multitenant
-
   belongs_to :issue
+  belongs_to :company
   belongs_to :watcher, polymorphic: true
 
   WATCHER_TYPE_USER = 'User'.freeze
```
Comment (Anmol79) [inline app/models/issue_watcher.rb]: should be a single constant WATECHER_TYPES = [ user: 'User', team: 'Team'].freeze

**Comment by basil7vals** [comment]
Cycle - 2 completed

**Comment by basil7vals** [comment]
Cycle - 2 completed

**Comment by basil7vals** [comment]
Final review changes done

**Pull request merged**